src/Keboola/DbExtractor/Extractor/Snowflake.php

```diff
@@ -393,7 +411,9 @@ public function getTables(?array $tables = null): array
         $tableNameArray = [];
         $tableDefs = [];
         foreach ($arr as $table) {
-            if (($this->schema && $table['schema_name'] !== $this->schema) || $table['schema_name'] === 'INFORMATION_SCHEMA') {
+            if (($this->schema && $table['schema_name'] !== $this->schema)
```

Comment:
I would wrap both:

```
if (
...first...
...second
) { 
```
Now it's inconsistent - "if" opening bracket is on same line, closing backet on new line

Reply:
lol, I think I had it like that and changed it because of sniff
![image](https://user-images.githubusercontent.com/569139/48148029-1c836380-e2b9-11e8-9d9c-7d66e71bedc1.png)